BossMod/BossModule/ZoneModuleRegistry.cs

```diff
@@ -26,9 +26,9 @@ static ZoneModuleRegistry()
                 Service.Log($"Zone module {t} has no ZoneModuleInfo attribute, skipping");
                 continue;
             }
-            if (_modulesByCFC.ContainsKey(attr.CFCID))
```

Comment:
btw this change is only here because this file fails the code style check otherwise